libs/pruviloj/Pruviloj/Core.idr

```diff
@@ -225,13 +225,19 @@ refine tm =
         countPi (RBind _ (Pi _ _) body) = S (countPi body)
         countPi _ = Z
 
+||| A helper to extract the type representation for term.
+getTTType : Raw -> Elab TT
+getTTType r = do
```

Comment:
This library doesn't use the "dangling `do`" style, but rather the "`do` is part of the expression" style. Please keep this consistent.